reference/compiling_for_android.rst

```diff
@@ -221,7 +194,9 @@ Application exits immediately
 If the application runs but exits immediately, there might be one of the
 following reasons:
 
--  libgodot_android.so is not in ``libs/armeabi``
+-  Make sure to use export templates that match your editor version; if
+   you use a new Godot version, you *have* to update the templates too.
+-  libgodot_android.so is not in ``lib/armeabi-v7a`` or ``lib/armeabi``
```

Comment:
I added `lib/armeabi` as I assume it's what will be used when building for armv5 instead of armv7.